autoscheduler/frontend/src/components/NavBar/NavBar.css

```
@@ -0,0 +1,4 @@
+#NameAndButton {
```

```suggestion
.name-and-button {
```
To be consistent with our other CSS rules, also since we're using classes instead of ids for like everything else on the site might as well conform to it